packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -86,12 +119,17 @@ export class FauxClassGenerator {
     }
   }
 
+  private capitalize(input: string): string {
+    return input.charAt(0).toUpperCase() + input.slice(1);
+  }
+
+  private getGenType(describeType: string): string {
```

Comment:
Do not use short names in method names. Expand `gen`

Reply:
I'll follow what you want for a style.  I personally find the sometimes very long names to be unreadable in practice...